README.rst

```diff
@@ -37,32 +37,34 @@
 
 |CI|  |COV| |PEP| |DOC| |VERS| |PY| |FORUM|
 
-`PennyLane <https://pennylane.readthedocs.io>`_ is a cross-platform Python library for quantum machine learning,
-automatic differentiation, and optimization of hybrid quantum-classical computations.
+`PennyLane <https://pennylane.readthedocs.io>`_ is a cross-platform Python library for **differentiable quantum programming**. Train a quantum computer the same way as a neural network.
+ 
+.. raw:: html
```

Comment:
I think what's is missing here is a short explanation of what PennyLane actually *does*. `docs/index.rst` has the same problem. If you read it without already knowing what PennyLane is, you'll only have a very vague idea about what it can do or which problem it solves. It could be something like

PennyLane lets you define parametrized quantum circuits (in both the qubit and the continuous-variable models), and encapsulate them into a *quantum node* object that behaves like a classical function. The quantum nodes can be evaluated using a number of popular simulator and quantum hardware backends, implemented using an extensible plugin architecture. Furthermore, the quantum nodes can be differentiated, and interfaced with popular autodifferentation frameworks, such as Autograd, PyTorch and TensorFlow.

This would be naturally followed by the next paragraph, which lists possible applications areas.

Reply:
Thanks @smite. We're trying something new with the readme (focus on simplicity, and still a WIP). I think your suggestion might be good for `docs/index.rst` though